packages/core/addon/serializers/table.js

```diff
@@ -23,12 +44,18 @@ export default VisualizationSerializer.extend({
   normalize(type, visualization) {
     if (visualization) {
       let columns = get(visualization, 'metadata.columns').map(column => {
+        let type = get(column, 'type'),
+          mapped = column;
+
         if (typeof get(column, 'field') === 'string') {
-          let type = get(column, 'type');
+          mapped = MAP_STRING_FIELD[type](column);
+        }
 
-          return TRANSFORMS[type](column);
+        if (isPresent(get(mapped, 'field'))) {
+          mapped = mapFieldToAttributes(type === 'threshold' ? 'metric' : type, mapped);
```

Comment:
Do we need to use `mapFieldToAttributes`? I feel like we should just change the functions within `MAP_STRING_FIELD` to return the right shape.

Reply:
Not against it, thought the change itself might be more clear if I just added another mapping on top of what was in place